feature/platform/transceiver/otg_test/zr_temperature_test/zr_temperature_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zr_temperature_test
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Topology:
+//   dut1:port1 <--> port1:dut2
+//
+
+func TestZRTemperatureState(t *testing.T) {
+	dut1 := ondatra.DUT(t, "dut1")
+	dp1 := dut1.Port(t, "port1")
```

Comment:
The interface is configured in the second test case, but not in this one; it should be done in both (unless you're relying on the specific test ordering, assuming that  TestZRTemperatureStateInterfaceFlap will be executed first)

Reply:
Hi Sergey,

Updated the script as suggested please review and let me know any further changes required.

Thanks